### PR TITLE
Development/notification

### PR DESCRIPTION
**Pull request by yacineKahlerras**
- notification popover
- notification page

### CODE REVIEW

**Comment by yacineKahlerras** [comment]
test test

File: packages/UI/src/components/organisms/Navigation/NotificationPopover/NotificationPopover.tsx
Comment (yacineKahlerras) [reply packages/UI/src/components/organisms/Navigation/NotificationPopover/NotificationPopover.tsx]: its been deleted and pushed

File: apps/website/src/pages/_app.js
Comment (yacineKahlerras) [reply apps/website/src/pages/_app.js:49]: i dont know @Enjoy2Live has made this change but i guess `<GlobalStyles/>` imports styled component css configs and its not needed anymore since we're using tailwind

File: packages/UI/src/components/organisms/Navigation/NotificationPopover/NotificationPopover.tsx
Comment (dbradham) [reply packages/UI/src/components/organisms/Navigation/NotificationPopover/NotificationPopover.tsx]: Thanks!

File: apps/website/src/pages/_app.js
Comment (dbradham) [reply apps/website/src/pages/_app.js:49]: Got it. So this change should not have any effect then?

File: apps/app/pages/_app.js
Comment (chungthuang) [inline apps/app/pages/_app.js:55]: What does `QueryClientProvider` do?

File: apps/website/src/pages/create.js
Comment (chungthuang) [inline apps/website/src/pages/create.js:18]: https://github.com/dev-launchers/dev-launchers-platform/pull/2110/files#r1826864273 sounds like we want to remove this because we migrated to tailwind. Why are we adding it back here?

File: apps/website/src/pages/learn.js
Comment (chungthuang) [inline apps/website/src/pages/learn.js:12]: https://github.com/dev-launchers/dev-launchers-platform/pull/2110/files#r1826886475

File: packages/UI/src/components/NotificationItem/NotificationItem.tsx
Comment (chungthuang) [inline packages/UI/src/components/NotificationItem/NotificationItem.tsx:160]: Should we add quote around `Commented`?

File: packages/UI/src/components/organisms/Navigation/NotificationPopover/NotificationPopover.tsx
Comment (chungthuang) [inline packages/UI/src/components/organisms/Navigation/NotificationPopover/NotificationPopover.tsx:15]: Do we need to pass this object? `agent.Notifications.get` calls `requests.get`, so method should already be set to GET.

File: packages/UI/src/components/organisms/Navigation/NotificationPopover/NotificationPopover.tsx
Comment (chungthuang) [inline packages/UI/src/components/organisms/Navigation/NotificationPopover/NotificationPopover.tsx:21]: Let's remove this if it's not used.

File: packages/tailwind-constructor/tailwind.config.js
Comment (yacineKahlerras) [reply packages/tailwind-constructor/tailwind.config.js:11]: there's no src folder under apps/app

File: packages/UI/src/components/NotificationItem/NotificationItem.tsx
Comment (yacineKahlerras) [reply packages/UI/src/components/NotificationItem/NotificationItem.tsx:160]: we dont need to

File: apps/website/src/pages/_app.js
Comment (yacineKahlerras) [reply apps/website/src/pages/_app.js:49]: no it shouldnt

Comment (dbradham) [reply apps/website/src/pages/_app.js:49]: thanks for confirming!

**Review by dbradham — APPROVED** [review]
Looks good!

**Comment by dbradham** [comment]
@Enjoy2Live @chungthuang @dev-launchers/universal-design can we please get your help to review this PR? It is blocked until we get approval from y'all. Thanks!

File: packages/UI/src/components/organisms/Navigation/NotificationPopover/NotificationPopover.tsx
Comment (chungthuang) [inline packages/UI/src/components/organisms/Navigation/NotificationPopover/NotificationPopover.tsx:90]: Just a note that we'll need to update this once we support notifications for other resources besides ideas.

File: packages/utility/agent.ts
Comment (chungthuang) [inline packages/utility/agent.ts:166]: This is fine for now, but @dbradham we should thrive to remove the need to use `populate=deep`.

**Comment by chungthuang** [comment]
Please squash into 1 commit.

File: packages/utility/agent.ts
Comment (dbradham) [reply packages/utility/agent.ts:166]: @chungthuang what is the ideal way to use `populate`? Should we use `populate=*`, or only using specific entities?

**Comment by dbradham** [comment]
@yacineKahlerras let us know if you have any blockers on the process to squash?

File: packages/utility/agent.ts
Comment (chungthuang) [reply packages/utility/agent.ts:166]: We should limit to specific entries. With `populate=deep`, strapi will populate nested relations. For example, each notification references a user, a user references a project, a project references an opportunity. `populate=deep` will return all those relations.

**Comment by chungthuang** [comment]
I pushed a new version that attempts to squash the commits into 1.

**Pull request merged**
